pageserver/src/tenant/storage_layer.rs

```diff
@@ -601,6 +603,13 @@ impl Drop for ValuesReconstructState {
 
 impl ValuesReconstructState {
     pub(crate) fn new(io_concurrency: IoConcurrency) -> Self {
+        Self::new_with_read_path(io_concurrency, None)
+    }
+
+    pub(crate) fn new_with_read_path(
+        io_concurrency: IoConcurrency,
+        read_path: Option<ReadPath>,
+    ) -> Self {
```

Comment:
This function is never called outside this impl. Does it make sense to just change `new` instead and always set `read_path = None`?